Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -2395,6 +2395,8 @@ public void HandlePacket(Client client, FriendRequestResponsePacket packet)
 
                     PacketSender.SendFriends(target);
                 }
+
+                player.FriendRequester = null;
```

Comment:
Perhaps we could just have _one_ `player.FriendRequester = null` at the end of this method if this implementation is safe.

Reply:
you are right, maybe we don't need to call this twice, i will give it a try by placing it elsewhere.

Reply:
Done ! 👍🏽